### DISCUSSION TITLE
Manage password issues

### DISCUSSION THREAD

**Discussion by chengyoufan** [requarks/wiki#7481] [Help / Questions]
忘记了管理密码，在登录页面点击“忘记密码”，输入正确的邮箱地址后点击下方“重置密码”，邮箱无法收到邮件。
请问如何才能重置或者找管理回密码？
现在能够打开postgresql数据库。

I forgot my management password. On the login page, click "Forgot Password", enter the correct email address, and then click "Reset Password" below. My email cannot receive emails.
How can I reset or retrieve my password from the administrator?
I can now open the PostgreSQL database.

![微信图片_20241215104726](https://github.com/user-attachments/assets/64c53306-142a-47fc-94ca-038a9b36b9a5)

**Comment by leosk-knight** [comment]
你应该没有设置邮件验证，我也是这种情况
https://github.com/requarks/wiki/discussions/7482
我尝试了用手册里边Troubleshooting提供的方法，没有成功，你刻意尝试一下
